tests/e2e/publish.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+TIME=$(date +%s)
+
+if [[ $CHE_VERSION == *SNAPSHOT ]] ; then 
+    echo "Building and pushing latest E2E package version $CHE_VERSION"
+    VERSION=${CHE_VERSION%"-SNAPSHOT"}-$TIME
+    TAG="nightly"
```

Comment:
"TAG" variable looks redundant now.

Reply:
Yeah, it's prepared for the https://github.com/eclipse/che/issues/18772 but I can comment it out.